### PR TITLE
chore: add credentuals-interface with lerna only, add and export inte…

### PR DESCRIPTION
**Pull request by whitneypurdum**
Node v. 16.10:
1. Add ew-registry "@ew-did-registry/credentials-interface": "0.6.3-alpha.600.0" using lerna command:

whitneypurdum@Whitneys-MacBook-Pro ew-credentials % lerna add @ew-did-registry/credentials-interface@0.6.3-alpha.600.0 --scope=@energyweb/credential-governance --dev
info cli using local version of lerna
lerna notice cli v4.0.0
lerna notice filter including "@energyweb/credential-governance"
lerna info filter [ '@energyweb/credential-governance' ]
lerna info Adding @ew-did-registry/credentials-interface in 1 package
lerna info Bootstrapping 3 packages
lerna info Symlinking packages and binaries
lerna success Bootstrapped 3 packages

3. Add and export interfaces
4. Update github workflow for lint command to use Node 16.10
5. Run install, setup build at root level
6. Check in all files including package-lock.json

### CODE REVIEW

File: .github/workflows/codelint.yml [hunk 1]
```diff
@@ -6,7 +6,10 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
```
[the remark this answers is not in the document]
Comment (whitneypurdum) [reply .github/workflows/codelint.yml:9]: Okay I’ll add it back in - should it be the first in the order you think?

**Review by jrhender — APPROVED** [review]
Looks good to me!

**Review by nichonien — APPROVED** [review]
Thanks @whitneypurdum! If it's feasible to clean up the commit history, it would be good or maybe squash and merge as one commit.

**Comment by whitneypurdum** [comment]
> Thanks @whitneypurdum! If it's feasible to clean up the commit history, it would be good or maybe squash and merge as one commit.

What do you usually do? git merge develop --squash?

**Pull request closed without merging**